src/app/SignalR/Startup.cs

```diff
@@ -124,7 +124,7 @@ private void ConfigureHealthChecks(IServiceCollection services)
             var dbSettings = new DbSettings();
             _configuration.GetSection("Db").Bind(dbSettings);
 
-            services.ConfigureHealthChecks(dbSettings);
+            services.ConfigureHealthChecks(dbSettings, new CacheSettings { IsEnabled = false });
```

Comment:
Potentially websocket server may also need to access the cache. Let's add Cache section to its settings and configure as in Api projects.